### DISCUSSION TITLE
ND display freezes with a red circle ..

### DISCUSSION THREAD

**Discussion by F-GER-CMD** [flybywiresim/aircraft#4977] [Help]
Hello,
my first post, so I hope this is the right way to feedback ..

Loaded a new flightplan from simbrief via aoc menue and init page, all setup done
Upon takeoff, left ND display freezes at 10 nm range and shows a red circle 
Intended or error?
Chaningng from ARC to PLAN does not resolve, yet changing range to 20nm refreshes NavigationDisplay
![ND_display-error](https://user-images.githubusercontent.com/85191727/120373849-222e7200-c319-11eb-9121-4b1f4944dbe3.jpg)

**Comment by ExampleWasTaken** [comment]
Looks like a mod conflict. Remove all other content from your community folder and try again.
